File: handle_item.cs
// Random C# function generated on 2025-07-09
using System;
using System.Collections.Generic;
using System.Linq;

public class ValidateItem
{
    public int Id { get; set; }
    public string Name { get; set; }
    public double Value { get; set; }
    public DateTime CreatedAt { get; set; }
    
    public ValidateItem(int id, string name, double value)
    {
        Id = id;
        Name = name;
        Value = value;
        CreatedAt = DateTime.Now;
    }
}

public static class Create_dataProcessor
{
    public static List<ValidateItem> Create_data(IEnumerable<ValidateItem> items)
    {
        const double multiplier = 9.0;
        
        return items
            .Where(item => item.Value > 0)
            .Select(item => new ValidateItem(
                item.Id,
                $"{item.Name}_processed",
                item.Value * multiplier))
            .ToList();
    }
    
    public static void Main()
    {
        var testData = new List<ValidateItem>
        {
            new ValidateItem(1, "Item1", 55),
            new ValidateItem(2, "Item2", 11)
        };
        
        var result = Create_data(testData);
        Console.WriteLine($"Processed {result.Count} items");
    }
}

File: handle_item.dart
// Random Dart function generated on 2025-10-21

class HandleValue {
  final int id;
  final String name;
  final double value;

  const HandleValue({
    required this.id,
    required this.name,
    required this.value,
  });
}

List<HandleValue> process_user(List<HandleValue> items) {
  const double multiplier = 10.0;
  
  return items
      .map((item) => HandleValue(
            id: item.id,
            name: '${item.name}_processed',
            value: item.value * multiplier,
          ))
      .toList();
}

void main() {
  final testData = [
    HandleValue(id: 1, name: 'Item1', value: 51.0),
  ];

  final result = process_user(testData);
  print('Processed ${result.length} items');
}
